core/network/src/protocol.rs

```diff
@@ -39,7 +41,11 @@ use crate::{error, util::LruHashSet};
 
 const REQUEST_TIMEOUT_SEC: u64 = 40;
 const TICK_TIMEOUT: time::Duration = time::Duration::from_millis(1000);
-const PROPAGATE_TIMEOUT: time::Duration = time::Duration::from_millis(5000);
+const PROPAGATE_TIMEOUT: time::Duration = time::Duration::from_millis(5100);
```

Comment:
(as a note you can also write `const PROPAGATE_TIMEOUT: time::Duration = time::Duration::from_millis(STATUS_INTERVAL_MS + 100);` to emphase the relativity of it.)